ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -147,7 +148,36 @@ function renderInformationImageRow(
   );
 }
 
-// TODO: test
+/**
+ * return true if markdown is loaded (description is rendered inside a markodown component)
+ * return true if description not exists but values are present
+ * return false in others cases
+ *
+ * this behavior is due to markdown loading: it could happen some items overlapped while the
+ * markdown content is loading. To avoid this, we wait the loading ends and then the items will
+ * be displayed
+ */
+export const canRenderItems = (
+  isMarkdownLoaded: boolean,
+  potServiceMetadata: ServiceMetadataState
+): boolean => {
+  if (
+    pot.isSome(potServiceMetadata) &&
+    potServiceMetadata.value &&
+    potServiceMetadata.value.description !== undefined
+  ) {
+    return isMarkdownLoaded;
+  } else if (
+    pot.isSome(potServiceMetadata) &&
+    potServiceMetadata.value !== undefined
+  ) {
+    return (
+      pot.isSome(potServiceMetadata) && potServiceMetadata.value !== undefined
```

Comment:
Did you note the **else if** statement is the same code of the **return**? 
You can just **return true**

Reply:
Could this function be rewritten as follows?

```typescript
export const canRenderItems = (
  isMarkdownLoaded: boolean,
  potServiceMetadata: ServiceMetadataState
): boolean => {
  const isServiceMetadataLoaded =
    pot.isSome(potServiceMetadata) && potServiceMetadata.value;
  const hasServiceMetadataDescription =
    pot.isSome(potServiceMetadata) && potServiceMetadata.value
      ? potServiceMetadata.value.description !== undefined
      : false;
  // if service metadata is loaded
  // return isMarkdownLoaded if it has a defined description field
  // return true otherwise
  if (isServiceMetadataLoaded) {
    return hasServiceMetadataDescription ? isMarkdownLoaded : true;
  }
  return false;
};
```